{"name": "test_login_with_OTP", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestEmbibeApp' object has no attribute 'getLogger'", "trace": "self = <Testcases.test_E2E.TestEmbibeApp object at 0x102957eb0>\n\n    @pytest.mark.usefixtures(\"appiumdriver\", \"log_on_failure\")\n    def test_login_with_OTP(self):\n        lp = landingpages(self.driver)\n        lp.signinusingOTP()\n>       log = self.getLogger()\nE       AttributeError: 'TestEmbibeApp' object has no attribute 'getLogger'\n\nTestcases/test_E2E.py:28: AttributeError"}, "start": 1732374462811, "stop": 1732374479302, "uuid": "a6493469-c455-4838-8c4e-2e02781bcbff", "historyId": "28339fab6a1ed379d621671bb4e90489", "testCaseId": "28339fab6a1ed379d621671bb4e90489", "fullName": "Testcases.test_E2E.TestEmbibeApp#test_login_with_OTP", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('appiumdriver', 'log_on_failure')"}, {"name": "parentSuite", "value": "Testcases"}, {"name": "suite", "value": "test_E2E"}, {"name": "subSuite", "value": "TestEmbibeApp"}, {"name": "host", "value": "EBL-C02H385BQ6L4.local"}, {"name": "thread", "value": "6402-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Testcases.test_E2E"}]}